tests/scripts/set-release-ver.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+FROM_TAG=$(grep "docker.io/rook/ceph" deploy/examples/images.txt | awk -F : '{ print $2 }')
+TO_TAG="$1"
+
+# This script is supposed to facilitate the preparation of a rook release by
+# changing example and documentation files to use the new release version instead the old one.
+#
+# It will usually be run on a release branch and the resulting changes will be the content of the
+# commit for the basis of the release.
+
+# The script uses two version tags:
+
+# FROM_TAG: the version tag of the previous/current release (e. g. v1.16.7)
+# TO_TAG: the version tag of the new release being prepared (e. g. v1.16.8)
+
+# TO_TAG should be passed to the script as the first positional parameter in its invocation.
+
+# So an example invocation could be:
+# set-release-ver.sh v1.17.2
+#
+# This assumes invocation from the release-1.17 branch with latest
+# release tag v1.17.1
+#
+# The script replaces all occurrences of $FROM_TAG with $TO_TAG in
+# the relevant affected files.
```

Comment:
Can this be summarized in fewer lines? 

I can see several things that can be cleaned up:
1. FROM_TAG is no longer an input and thus doensn't need documentation
2. invocation isn't needed because it's documented in the release  readme
3. don't need to state TO_TAG is final positional because that's implicitly documented above

In addition, just some repetition that can be reduced. I think this could be 2-3 lines and still perfectly helpful

Reply:
done.